src/components/toolbox/passphraseInput/passphraseInput.css

```
@@ -1,24 +1,72 @@
 @import '../../../app/variables.css';
 @import '../../../app/mixins.css';
 
+.wrapper {
+  margin: 0;
+  position: relative;
+}
+
 .inputs {
+  width: 100%;
+  display: flex;
   flex-wrap: wrap;
+  flex-direction: row;
+  border: 1px solid var(--dark-border-color);
+  border-radius: 3px;
+  background-color: var(--color-white);
+  padding: 0px 24px;
+  box-sizing: border-box;
+  outline: none;
+
+  &:focus-within {
+    border-color: var(--color-ultramarine-blue);
+  }
 
   & > span {
-    margin: 0 0 12px 0;
+    width: calc(100% / 6);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    flex-grow: 1;
+    box-sizing: border-box;
+    color: var(--color-ghost);
+
+    &:focus-within {
+      color: var(--color-ultramarine-blue);
+    }
+
+    & .inputNumber {
+      width: 20px;
+    }
+
+    & .inputNumberError {
+      color: var(--color-burnt-sienna);
+    }
 
     & input {
-      text-align: center;
+      border: none;
+      text-align: left;
+      padding: 0 10px !important;
+
+      &:focus {
+        color: var(--color-ultramarine-blue);
+      }
     }
   }
 }
 
+.boxOnError {
+  border-color: var(--color-burnt-sienna) !important;
```

💅 Better to avoid `important`:
```suggestion
.boxOnError,
.boxOnError:focus-within {
  border-color: var(--color-burnt-sienna);
```

---

You added `boxOnError:focus-within ` but still didn't remove `!important` @osvaldovega